ansible/roles/anitya-dev/tasks/main.yml

```
@@ -37,7 +37,7 @@
   become_user: "{{ ansible_env.SUDO_USER }}"
   shell: >
       source ~/.bashrc && mkvirtualenv anitya -a ~/devel/ &&
-      virtualenv-3 --system-site-packages ~/.virtualenvs/anitya/
+      virtualenv --system-site-packages ~/.virtualenvs/anitya/
```

What is the purpose of this change?

---

It would be good to be explicit about using Python 3:

```suggestion
      virtualenv --python=/usr/bin/python3 --system-site-packages ~/.virtualenvs/anitya/
```